flang/include/flang/Semantics/openmp-directive-sets.h

```diff
@@ -194,6 +194,7 @@ static const OmpDirectiveSet allDistributeParallelDoSimdSet{
 static const OmpDirectiveSet allDistributeSimdSet{
     allDistributeSet & allSimdSet};
 static const OmpDirectiveSet allDoSimdSet{allDoSet & allSimdSet};
+static const OmpDirectiveSet scanAllowedSet{allDoSet | allSimdSet};
```

Comment:
I'd suggest placing this into a new section in this header, since it doesn't belong in any of the ones that already exist. Maybe call the variable `scanParentAllowedSet` and the section "Directive sets for allowed/not allowed parent directives" or something like this.

Reply:
Made the change. Thank you for the suggestion.